src/shared/types/IlsNavaid.ts

```diff
@@ -1,12 +1,13 @@
-import { DatabaseItem, Degrees, DegreesMagnetic, KiloHertz, Location, LsCategory } from './Common';
+import { Coordinates, Degrees, DegreesMagnetic } from 'msfs-geo';
+import { DatabaseItem, KiloHertz, LsCategory } from './Common';
 
 export interface IlsNavaid extends DatabaseItem {
     frequency: KiloHertz;
     category: LsCategory;
     runwayIdent: string;
-    locLocation: Location;
+    locLocation: Coordinates;
     locBearing: DegreesMagnetic;
-    gsLocation?: Location;
+    gsLocation?: Coordinates;
```

Comment:
Needs to be ElevatedCoordinates | Coordinates.

Reply:
we can just set the elevation to the airport elevation if the ils elevation isnt defined right?